### PR TITLE
add index file

### PR DESCRIPTION
**Pull request by ryanxmccarthy**
Best index file ever

### CODE REVIEW

**Comment by ppreyer** [comment]
This is weak code.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
```
Comment (ppreyer) [inline index.html:2]: Comment left.

**Comment by ryanxmccarthy** [comment]
I would like to quit.

File: index.html (hunk 1 above)
Comment (ppreyer) [inline index.html:2]: starting a review...

**Review by ppreyer — APPROVED** [review]
Good stuff.

**Pull request merged**